Jenkinsfile

```
@@ -51,7 +51,7 @@ node ('lagoon-images') {
         }
 
         stage ('Copy examples down') {
-          sh script: "git clone https://github.com/uselagoon/lagoon-examples.git tests"
+          sh script: "git clone --branch new_repos https://github.com/uselagoon/lagoon-examples.git tests"
```

```suggestion
          sh script: "git clone https://github.com/uselagoon/lagoon-examples.git tests"
```
once branch is merged in lagoon-examples